src/main.rs

```diff
@@ -66,25 +70,39 @@ async fn main() -> TofndResult<()> {
     let main_span = span!(Level::INFO, "main");
     let _enter = main_span.enter();
 
-    let incoming = TcpListener::bind(addr(cfg.port)).await?;
+    let incoming_gg20 = TcpListener::bind(addr(cfg.gg20_port)).await?;
+    let incoming_multisig = TcpListener::bind(addr(cfg.multisig_port)).await?;
     info!(
-        "tofnd listen addr {:?}, use ctrl+c to shutdown",
-        incoming.local_addr()?
+        "use ctrl+c to shutdown\ntofnd listen addresses:\n\tGG20: {:?}\n\tmultisig: {:?}, ",
```

Comment:
```suggestion
        "use ctrl+c twice to shutdown\ntofnd listen addresses:\n\tGG20: {:?}\n\tmultisig: {:?}, ",
```
Since the user needs it twice for now.